pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-konvoy/enterprise/index.md

```diff
@@ -80,7 +80,15 @@ Your cluster comes preconfigured with a few different core addons that provide f
 
 <p class="message--warning"><strong>IMPORTANT:</strong>Ensure your <code>dkp</code> configuration references the management cluster where you want to run the upgrade by setting the <code>KUBECONFIG</code> environment variable, or using the <code>--kubeconfig</code> flag, <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">in accordance with Kubernetes conventions</a>.
 
-Replace `my-aws-cluster` with the name of the cluster.
+Upgrade the core addons in a cluster with one of the choices [aws, azure, preprovisioned] by replacing `my-provider-cluster` with the desired cluster name.
+
+Examples:
+
+```bash
+export CLUSTER_NAME=my-azure-cluster
+dkp upgrade addons azure --cluster-name=${CLUSTER_NAME}
+```
+OR
```

Comment:
I think this is ok, but then below we should change the "The output should be similar to" below -->  "The output for an AWS cluster should be similar to:"

Reply:
Will do